src/network/network_content.cpp

```diff
@@ -743,6 +743,7 @@ class NetworkContentConnecter : TCPConnecter {
 	void OnConnect(SOCKET s) override
 	{
 		assert(_network_content_client.sock == INVALID_SOCKET);
+		_network_content_client.lastActivity = _realtime_tick; // no matter how long it took to connect, we can start sending now.
```

Comment:
`count being able to connect as activity` ?

But honestly, removing the comment works fine for me too :)

Reply:
to me it btw reads that this line can now be removed:

https://github.com/OpenTTD/OpenTTD/pull/8530/files#diff-08315443067917cc78b4d1d25d91f79676da81013e66a51009a2abbac480361bR759

As `SendReceive()` is aborted if the socket is still connecting, and as such can never run the activity check. Makes the code a bit more undestandable too I guess, and why this PR is needed :D